web/src/js/urlState.js

```diff
@@ -78,6 +78,6 @@ function updateUrlFromStore(store) {
 }
 
 export default function initialize(store) {
-    updateStoreFromUrl(store)
     store.subscribe(() => updateUrlFromStore(store))
+    updateStoreFromUrl(store)
```

Comment:
![default](https://cloud.githubusercontent.com/assets/9366279/25564571/bb4ddbf8-2de8-11e7-8b1d-675773f58112.png)
Here's the coverage report before the change, seems that it could not hit `updateUrlFromStore.`
I am confused about why this happen, but changing the order of the code can fix this.